### PR TITLE
added new level from Max

### PR DESCRIPTION
**Pull request by stallop**

### CODE REVIEW

**Review by lsizemore8 — APPROVED** [review]
Resolve the conflict but I like it!

File: core/assets/jsons/save.json [hunk 1]
```diff
@@ -1,50 +1,57 @@
...
+  "levels": [
+    {
+      "name":       "easyLevel",
+      "unlocked":   true,
+      "completed":  true,
+      "path":       "levels/easyLevel.json"
+    },
+    {
+      "name":       "mediumLevel",
+      "unlocked":   true,
```
Comment (BryanGraeser) [inline core/assets/jsons/save.json]: This file should NOT be set to all unlocked. This determines what is initially unlocked in the JAR

**Review by BryanGraeser — CHANGES_REQUESTED** [review]
`core/assets/jsons/save.json` should not have all files unlocked.

File: core/assets/jsons/save.json (hunk 1 above)
Comment (BryanGraeser) [reply core/assets/jsons/save.json]: You should add your level, but leave everything locked except the first tutorial level

**Review by BryanGraeser — APPROVED** [review]
fixed save issues. gtg

**Pull request merged**